competitions/cloud-cover/benchmark-tutorial.ipynb

```diff
@@ -0,0 +1,2586 @@
+       "      <th>B04_path</th>\n",
+       "      <th>B08_path</th>\n",
+       "      <th>label_path</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>6571</th>\n",
+       "      <td>pbyl</td>\n",
+       "      <td>Lodwar</td>\n",
+       "      <td>2020-09-08 08:09:15+00:00</td>\n",
+       "      <td>az://./train_features/pbyl</td>\n",
+       "      <td>2020</td>\n",
+       "      <td>/driven-data/cloud-cover/train_features/pbyl/B...</td>\n",
+       "      <td>/driven-data/cloud-cover/train_features/pbyl/B...</td>\n",
+       "      <td>/driven-data/cloud-cover/train_features/pbyl/B...</td>\n",
+       "      <td>/driven-data/cloud-cover/train_features/pbyl/B...</td>\n",
+       "      <td>/driven-data/cloud-cover/train_labels/pbyl.tif</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     chip_id location                  datetime                   cloudpath  \\\n",
+       "6571    pbyl   Lodwar 2020-09-08 08:09:15+00:00  az://./train_features/pbyl   \n",
+       "\n",
+       "      year                                           B02_path  \\\n",
+       "6571  2020  /driven-data/cloud-cover/train_features/pbyl/B...   \n",
+       "\n",
+       "                                               B03_path  \\\n",
+       "6571  /driven-data/cloud-cover/train_features/pbyl/B...   \n",
+       "\n",
+       "                                               B04_path  \\\n",
+       "6571  /driven-data/cloud-cover/train_features/pbyl/B...   \n",
+       "\n",
+       "                                               B08_path  \\\n",
+       "6571  /driven-data/cloud-cover/train_features/pbyl/B...   \n",
+       "\n",
+       "                                          label_path  \n",
+       "6571  /driven-data/cloud-cover/train_labels/pbyl.tif  "
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "example_chip = train_meta[train_meta[\"chip_id\"] == \"pbyl\"]\n",
+    "\n",
+    "display(example_chip)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The example chip is an image taken from Lodwar, a northwestern town in Kenya."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'driver': 'GTiff',\n",
+       " 'dtype': 'uint16',\n",
+       " 'nodata': 0.0,\n",
+       " 'width': 512,\n",
+       " 'height': 512,\n",
+       " 'count': 1,\n",
+       " 'crs': CRS.from_epsg(32636),\n",
+       " 'transform': Affine(10.0, 0.0, 771935.0,\n",
+       "        0.0, -10.0, 331300.0)}"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "example_chip = example_chip.iloc[0]\n",
```

Comment:
Can you avoid reassigning this to the same variable? Previously this was a dataframe and now it's becoming a series, so users wouldn't be able to rerun this cell.

Reply:
great point!